### PR TITLE
video upload help

### PR DESCRIPTION
**Pull request by Muddasser**

### CODE REVIEW

**Comment by Muddasser** [comment]
@raeeschachar @benpatterson @muhammad-ammar Please review. 

This PR tests Help link on video upload page on studio side.

File: regression/pages/studio/video_upload_studio.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    Studio video upload page.
+    """
+
+    url_path = "videos"
+
+    def is_browser_on_page(self):
+        """
+        Returns whether browser is on the page.
+        """
+        return self.q(css='body.view-video-uploads').present
```
Comment (benpatterson) [inline regression/pages/studio/video_upload_studio.py]: should this be set to `.visible`?

File: regression/tests/studio/test_video_upload_help.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    Test help link on video upload page.
+    """
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(TestVideoUploadHelp, self).setUp()
+
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
```
Comment (benpatterson) [inline regression/tests/studio/test_video_upload_help.py]: nit: I'd group things like this:
```
declare page objects
...
do things with page objects
```

File: regression/tests/studio/test_video_upload_help.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        Scenario: Help link in navigation bar is working on
+        'Upload video' page.
+        Given that I am on the 'Upload video' page.
+        And I want help about the it.
+        And I click the 'Help' in the navigation bar
+        Then Help link should open.
+        And help url should contain 'video/video_uploads.html'
+        """
+        # The url we want to see in anchor help element.
+        href = 'https://edx.readthedocs.io/projects/edx-partner-course-staff/' \
```
Comment (benpatterson) [inline regression/tests/studio/test_video_upload_help.py]: nit: i'd call this `expected_href` to make it clearer

**Comment by benpatterson** [comment]
👍

**Comment by raeeschachar** [comment]
👍
Squash commits

**Pull request merged**